chat/src/main/java/greencity/service/impl/AzureFileServiceImpl.java

```diff
@@ -71,4 +60,19 @@ private BlobContainerClient containerClient() {
             .connectionString(connectionString).buildClient();
         return serviceClient.getBlobContainerClient(containerName);
     }
+
+    private ChatFileDto downloadFile(MultipartFile multipartFile, String filename) {
```

Comment:
Why the method name is 'download'? The method uploads files. Also, you throw FileNotSavedException, so the name of the method should be related to the 'saving'